Makefile

```diff
@@ -981,8 +981,14 @@ test-e2e: $(GINKGO) generate-e2e-templates ## Run the end-to-end tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: build-kind-test-images
+build-kind-test-images: ## Build the corresponding kindest/node images required for e2e testing
+	scripts/ci-build-kind.sh
+
 .PHONY: tilt-up
-tilt-up: kind-cluster ## Start tilt and build kind cluster if needed.
+tilt-up: kind-cluster ## Start tilt, build kind cluster, generate e2e templates and kindest/node images if needed.
+	$(MAKE) build-kind-test-images
+	$(MAKE) generate-e2e-templates
```

Comment:
Same as above, I don't want to wait for generate-e2e-templates every time I'm using tilt for interactive development